### DISCUSSION TITLE
Support SuperBoosters

### DISCUSSION THREAD

**Discussion by Sikatsu** [Zrips/CMI#4468] [Feature Requests]
**Description of feature/improvement request:**

---

I am wondering if you can support https://www.mc-market.org/resources/8505/, so if players use /sell on for example fish, it would be boosted / doubled as well.

**Comment by nitewing76** [comment]
Just as SuperBoosters added support for WildTools, ShopGUI+, and others...they would need to add support for CMI.  If _Plugin A_ wants to use or change something in _Plugin B_, then _Plugin A_ creates the hook. Since SuperBoosters wants to change CMI's shop prices, then SuperBoosters is _Plugin A_.

**Comment by Sikatsu** [comment]
Well, that's true, but the developer isn't planning to add support for more plugins. Instead I was advised that plugins add support for SuperBoosters instead.

**Comment by nitewing76** [comment]
I don't doubt it. He is arrogant and lazy. He has stated many times he will only add whatever features that he "wants to" (and does not seem to care how many ppl want a feature). It is more efficient if SB creates the hook, otherwise CMI will have to check SB every time someone uses /sell.  Use ShopGUI+...imho it's worth the $$ and works with SB (and other plugins that only hook into ShopGUI+).
